package File_handling;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.FileOutputStream;
import java.io.IOException;

public class Program5 {
	public static void main(String[] args)throws IOException
	{
		File f1=new File("Documents:\\cde.txt");
		FileOutputStream fos=null;
		try {
			fos=new FileOutputStream(f1);
			System.out.println("File is created and now u can write into it");
			fos.write(65);
			fos.write('B');
			fos.write("abc".getBytes());
		}
		catch(FileNotFoundException e)
		{
			System.out.println("File is not created");
		}
		finally {
			fos.close();
		}
		System.out.println("=====program ends==========");
	}

}
